### DISCUSSION TITLE
[Help]: Is there any way to stop when dragging and not letting go, switching to the next or previous one?

### DISCUSSION THREAD

**Discussion by guangzan** [davidjerleke/embla-carousel#892] [Help] [answered]
### Summary

Embla is an excellent library👍. I would like to ask, is there any way to achieve this effect? When I keep dragging and not letting go, it will keep switching to the next one. I want it to switch one and stop.

This is the target effect.

https://github.com/davidjerleke/embla-carousel/assets/43458667/26fdf2bc-9fcb-489a-b196-978704c6b5e0

This is the Embla Carousel effect.

https://github.com/davidjerleke/embla-carousel/assets/43458667/07db6d11-4c4e-4ac3-b4f1-6ac6cdbba0ef

### If applicable, which variants of Embla Carousel are relevant to this question?

- [X] embla-carousel **(Core)**
- [X] embla-carousel-react
- [ ] embla-carousel-vue
- [ ] embla-carousel-svelte
- [ ] embla-carousel-autoplay
- [ ] embla-carousel-auto-scroll
- [ ] embla-carousel-solid
- [ ] embla-carousel-auto-height
- [ ] embla-carousel-class-names
- [ ] embla-carousel-fade
- [ ] embla-carousel-docs **(Documentation)**
- [ ] embla-carousel-docs **(Generator)**

**Comment by davidjerleke** [comment]
@guangzan do you mean that the carousel should terminate the dragging and scroll to the next/closest slide? And the user has to grab the carousel with the pointer again if they want to drag it further?

**Comment by guangzan** [comment]
Hi, @davidjerleke,

Thanks for your reply. Yes, your description is even more accurate.

**Reply by davidjerleke** [reply, marked as answer]
@guangzan thanks for your question. The answer is no, Embla is opinionated in this matter. The reasoning here is that releasing the pointer when the user didn't intend to release it is bad UX and harms accessibility. How do we know that the user didn't mean to scroll further?
